cql3/statements/create_index_statement.cc

```diff
@@ -381,6 +382,15 @@ std::optional<create_index_statement::base_schema_with_new_index> create_index_s
                     format("Index {} is a duplicate of existing index {}", index.name(), existing_index.value().name()));
         }
     }
+    auto existing_vector_index = _properties->custom_class && _properties->custom_class == "vector_index" && secondary_index::vector_index::has_vector_index_on_column(*schema, targets[0]->column_name());
```

Comment:
Style nitpick: Please avoid using "auto" when it doesn't offer any advantage like avoiding to write a long type name that already appears on the RHS, or when it hides an ugly and confusing type name. Here you just used "auto" as a shorthand for "bool"...